src/components/member/tabs/quote-tab/quote-activation.tsx

```diff
@@ -104,54 +98,26 @@ export const QuoteActivation: React.FC<{
             <div>
               <strong>Activation date</strong>
             </div>
-            <div>
-              <DateTimePicker
-                date={activeFrom || new Date()}
-                setDate={(value) => {
-                  if (onWipChange) {
-                    onWipChange(true)
-                  }
-                  setActiveFrom(value)
-                }}
-              />
-            </div>
-          </BottomSpacerWrapper>
 
-          <BottomSpacerWrapper>
-            <Checkbox
-              onChange={(_, { checked }) => {
-                if (onWipChange) {
-                  onWipChange(true)
-                }
-                if (!checked) {
-                  setPreviousAgreementActiveTo(null)
-                }
-                setUseGap(!!checked)
-              }}
-              label="Create gap between insurances"
-              checked={useGap}
-            />
-          </BottomSpacerWrapper>
-
-          {useGap && (
-            <BottomSpacerWrapper>
+            {contract.isTerminated && (
```

Comment:
We will move away from `isTerminated` eventually
```suggestion
            {contract.terminationDate && (
```